msctl

```diff
@@ -2487,7 +2497,7 @@ case "$COMMAND" in
         if isWorldEnabled "$arg"; then
           WORLDS="$WORLDS $arg"
         else
```

Comment:
Maybe we should add in an `elif isWorldDisabled "$arg"; then` section..

Reply:
> Maybe we should add in an `elif isWorldDisabled "$arg"; then` section..

that could be a good addition. do we wanna do it for just the start/stop/restart commands or for all of the commands?

Reply:
I was just thinking about what was already being touched here. Instead of adding "or disabled", I thought it would be nicer to report separate error messages.

Reply:
ok, i just added commit a0c0f6d that adds the notifications for trying to start/stop/restart disabled worlds

Reply:
do we want the `force-stop` and `force-restart` commands to skip this check like @zanix eluded to?